tests/integration/test_groupby.py

```diff
@@ -251,6 +251,58 @@ def test_aggregations(c):
     assert_eq(return_df.reset_index(drop=True), expected_df)
 
 
+def test_stddev(c):
```

Comment:
Could we have some tests that run on gpu tables as well? 
Could we also add tests with multiple aggregations, and aggregations without groupby. 
Even if the multi agg ones fail because of #655 we can comment that out/mark as xfail linking to that issue